front/src/components/boxs/chart/ApexChartTimelineOptions.js

```diff
@@ -0,0 +1,215 @@
+const addYAxisStyles = () => {
+  const yAxisLabel = document.querySelectorAll('.apexcharts-yaxis-label');
```

Comment:
Where is this coming from?

ApexChart documentation?

Reply:
No, I found this solution on stackoverflow and on apexcharts github